plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/LocalDataTab.tsx

```diff
@@ -0,0 +1,88 @@
+import { CatalogItem, convertRaw } from "@web/extensions/sidebar/core/processCatalog";
+import { Form } from "@web/sharedComponents";
+import { InboxOutlined } from "@web/sharedComponents/Icon/icons";
+import Upload, { UploadProps, UploadFile } from "@web/sharedComponents/Upload";
+import { RcFile } from "antd/lib/upload";
+import { useCallback, useState } from "react";
+
+import FileTypeSelect from "./FileTypeSelect";
+
+type Props = {
+  onOpenDetails?: (data?: CatalogItem) => void;
+};
+
+const LocalDataTab: React.FC<Props> = ({ onOpenDetails }) => {
+  const [fileList, setFileList] = useState<UploadFile[]>([]);
+  const fileFormats = ".kml,.czml,.topojson,.geojson,.json,.gltf,.glb";
```

Comment:
You can move this outside of the LocalDataTab component since it'll never change

Reply:
Done!